aeron-driver/src/main/c/CMakeLists.txt

```diff
@@ -45,6 +51,10 @@ if(UUID_H_EXISTS)
     add_definitions(-DHAVE_UUID_H)
 endif()
 
+if(MSVC AND "${CMAKE_SYSTEM_NAME}" MATCHES "Windows")
+	set(AERON_LIB_WINSOCK_LIBS wsock32 ws2_32 Iphlpapi)	
```

Comment:
which functions are needed from Iphlpapi?

Reply:
It is needed to replace `getifaddrs`

It uses theses functions:
`if_nametoindex `
`GetAdaptersAddresses `
`ConvertLengthToIpv4Mask `